client/state/comments/reducer.js

```diff
@@ -133,6 +139,7 @@ export function requests( state = {}, action ) {
 
 	return state;
 }
+requests.schema = requestsSchema;
```

Comment:
Is request status interesting to store? Can we end up in an invalid state if we restore this information on the next page load?